moped-database/migrations/1741914333559_alter_table_public_moped_user_events_add_check_constraint_moped_user_events_event_name_check/up.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE moped_user_events DROP CONSTRAINT moped_user_events_event_name_check;
+
+ALTER TABLE moped_user_events
+ADD CONSTRAINT moped_user_events_event_name_check CHECK (event_name IN ('app_load', 'dashboard_load', 'projects_map_load, projects_saved_view'));
```

Comment:
there are a couple of missing single quotes that are turning the last values in this check into one. I tested with them, and this works 100%. Could you add those in?

Reply:
yes, thank you for noticing that!